pages/account_and_service_management/account_information/security-specification-pci/meta.yaml

```
@@ -0,0 +1,2 @@
+id: ec84433e-5187-4642-8e73-310292eaf82e
+full_slug: Public-cloud-instance-security-overview
```

```suggestion
full_slug: public-cloud-instance-security-overview
```
Upper case is not a good practice for SEO